Editor/EditorCore/UnityScenePostProcessor.cs

```diff
@@ -87,6 +92,9 @@ public static void OnPostprocessScene()
                 StripProBuilderScripts.DestroyProBuilderMeshAndDependencies(gameObject, mesh, true);
 
             }
+
+            foreach (var renderer in renderersToStrip)
+                Undo.DestroyObjectImmediate(renderer);
```

Comment:
Had to move this here as otherwise I'm getting errors in the `foreach (var mesh in pbMeshes)` loop above.